<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogRuleDefine</class>
 <widget class="QDialog" name="DialogRuleDefine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DialogRuleDefine{
border-image: url(:/widget/调距桨背景.png);
}

QGroupBox{
	background-color: rgba(16, 57, 70, 1);
	border: 0px solid rgb(111, 255, 255);
	margin-top:3.3ex;
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QGroupBox::title{
	subcontrol-origin: margin;
	subcontrol-position: top center;
	background-color: rgba(19, 67, 79, 1);
	padding: 7 1000px;
}

QLineEdit{
	background-color: rgba(255, 255, 255,0.2);
	border-width:0;
	border-style:outset;
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QLabel{
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QRadioButton{
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QRadioButton::indicator{
	border-color: rgba(255, 255, 255,0);
	background:transparent;
}


QRadioButton::indicator:checked{
	border-image:url(:/widget/images/green.png)
}

QRadioButton::indicator:unchecked{
	border-image:url(:/widget/images/unchecked.png)
}

QPushButton{
	border: 2px solid rgba(61, 255, 255,0.4);
	background-color: rgba(12, 34, 45, 0.6);
	font: 12pt &quot;微软雅黑&quot;;
	color: rgba(102, 249, 247, 1);
	text-align：center;
}

QPushButton::disabled{
	border: 0px;
	color: rgba(102, 249, 247, 0.3);
	text-align：center;
}

QPushButton::pressed{
	border: 2px solid rgba(61, 255, 255,0.8);
	background-color: rgba(12, 34, 45, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QPushButton::hover{
	border: 2.5px solid rgba(61, 255, 255,0.8);
	background-color: rgba(12, 34, 45, 0.6);
	font: 75 12pt &quot;微软雅黑&quot;;
}

QDoubleSpinBox{
	border: 0px solid rgba(61, 255, 255,0.4);
	background-color: rgba(255, 255, 255, 0.2);
	font: 12pt &quot;Times New Roman&quot;;
	color: rgba(102, 249, 247, 1);
	text-align：center;
	padding-left:10px;
}


QTabWidget{
	background-color: rgba(19, 67, 79, 1);
	border-color: rgba(19, 67, 79, 1);
	border:0px;
	margin:0px;
}

QTabWidget::pane{
	border:0px;
	margin:0px;
}

QTabWidget::tab-bar {
	left: 2px;
	border:0px;
	margin:0px;
}

QTabBar::tab:first {
	background-color: rgba(11, 49, 61, 1);
	border-color: rgba(11, 49, 61, 1);
	color: rgba(102, 249, 247, 1);
	min-width: 4em;
	min-height:1em;
}
QTabBar::tab {
	font: 12pt &quot;微软雅黑&quot;;
	background-color: rgba(11, 49, 61, 255);
	border-color: rgba(11, 49, 61, 255);
	color: rgba(102, 249, 247, 1);
	min-width: 4em;
	min-height:1em;
}

QTabBar::tab:first:selected {
	min-width: 4em;
	min-height:1em;
}

QTabBar::tab:selected {
margin-bottom:0.2em;
}

QTabBar::tab:!selected{
	min-width: 4em;
	min-height:1em;
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="4">
    <widget class="QGroupBox" name="groupHydraulic_14">
     <property name="minimumSize">
      <size>
       <width>1200</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
	background-color: rgba(16, 57, 70, 1);
	border: 0px solid rgb(111, 255, 255);
	margin-top:3.3ex;
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QGroupBox::title{
	subcontrol-origin: margin;
	subcontrol-position: top center;
	background-color: rgba(19, 67, 79, 1);
	padding: 7 1000px;
}

QLineEdit{
	background-color: rgba(255, 255, 255,0.2);
	border-width:0;
	border-style:outset;
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QLabel{
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QRadioButton{
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QRadioButton::indicator{
	border-color: rgba(255, 255, 255,0);
	background:transparent;
}


QRadioButton::indicator:checked{
	border-image:url(:/widget/images/green.png)
}

QRadioButton::indicator:unchecked{
	border-image:url(:/widget/images/unchecked.png)
}

QPushButton{
	border: 2px solid rgba(61, 255, 255,0.4);
	background-color: rgba(12, 34, 45, 0.6);
	font: 12pt &quot;微软雅黑&quot;;
	color: rgba(102, 249, 247, 1);
	text-align：center;
}

QPushButton::disabled{
	border: 0px;
	color: rgba(102, 249, 247, 0.3);
	text-align：center;
}

QPushButton::pressed{
	border: 2px solid rgba(61, 255, 255,0.8);
	background-color: rgba(12, 34, 45, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QPushButton::hover{
	border: 2.5px solid rgba(61, 255, 255,0.8);
	background-color: rgba(12, 34, 45, 0.6);
	font: 75 12pt &quot;微软雅黑&quot;;
}

QDoubleSpinBox{
	border: 0px solid rgba(61, 255, 255,0.4);
	background-color: rgba(255, 255, 255, 0.2);
	font: 12pt &quot;Times New Roman&quot;;
	color: rgba(102, 249, 247, 1);
	text-align：center;
	padding-left:10px;
}
</string>
     </property>
     <property name="title">
      <string>变量列表</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_12">
      <property name="topMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget_Variable">
        <property name="styleSheet">
         <string notr="true">

background-color: rgba(255, 255, 255,0.2);</string>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>基础变量</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableWidget" name="tableWidget_BasicVariable"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_9">
         <attribute name="title">
          <string>中间变量</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableWidget" name="tableWidget_MidVariable"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_10">
         <attribute name="title">
          <string>故障变量</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableWidget" name="tableWidget_FaliureVariable"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_11">
         <attribute name="title">
          <string>报警变量</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableWidget" name="tableWidget_WarnningVariable">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupHydraulic_11">
     <property name="minimumSize">
      <size>
       <width>370</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>基础信息</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <widget class="QLabel" name="label_30">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>规则名称：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_RuleName">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	background-color: rgba(255, 255, 255,1);
	border-width:0;
	border-style:outset;
	color: rgb(0, 0, 0);
	font: 12pt &quot;微软雅黑&quot;;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_18">
        <item>
         <widget class="QLabel" name="label_31">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>规则描述：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_RuleDescription">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	background-color: rgba(255, 255, 255,1);
	border-width:0;
	border-style:outset;
	color: rgb(0, 0, 0);
	font: 12pt &quot;微软雅黑&quot;;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_37">
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>规则位置：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ComboBox_RulePos">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string>ALL</string>
          </property>
          <item>
           <property name="text">
            <string>ALL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>液压系统</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>中心控制箱</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>机旁控制箱</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>备用泵电机启动箱</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>提升泵电机启动箱</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item>
         <widget class="QLabel" name="label_33">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>规则状态：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="modelCmb_State">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string>禁用</string>
          </property>
          <item>
           <property name="text">
            <string>禁用</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>启用</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_38">
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>编辑人：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Editor">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	background-color: rgb(150, 150, 150);
	border-width:0;
	border-style:outset;
	color: rgb(0, 0, 0);
	font: 12pt &quot;微软雅黑&quot;;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupHydraulic_13">
     <property name="minimumSize">
      <size>
       <width>370</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>规则定义</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="topMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QLabel" name="label_35">
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Condition：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_Condition"/>
      </item>
      <item>
       <widget class="QLabel" name="label_36">
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Then：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_Then"/>
      </item>
      <item>
       <widget class="QLabel" name="label_34">
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Else：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_Else"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string> 持续时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="SpinBoxDurTime">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string> 秒</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_21">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item>
      <widget class="QPushButton" name="Btn_Check">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>语法检查</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_OK">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_Cancel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
